ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+Playlist.swift

```diff
@@ -3,13 +3,15 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+import BraveCore
```

Comment:
nit: Not necessary ?

Reply:
You're right this was added because before I was checking `kNewPlaylistUI` feature flag in this file before, no longer needed

Reply:
this was cleaned up in one of the recent commits